packages/@webex/plugin-meetings/test/unit/spec/personal-meeting-room/personal-meeting-room.js

```diff
@@ -1,7 +1,7 @@
 /*!
  * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
  */
-
+import 'jsdom-global/register';
 import 'jsdom-global/register';
```

Comment:
Added redundantly.

Reply:
@rarajes2 thanks for comment, will update